src/main/java/com/google/research/bleth/servlets/ListSimulationsServlet.java

```diff
@@ -34,11 +35,20 @@
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Gson gson = new Gson(); // Used for json serialization.
 
-        // Read SimulationMetadata as a HashMap.
-        ImmutableMap<String, SimulationMetadata> simulations = SimulationMetadata.listSimulations();
+        // Read SimulationMetadata as a sorted LinkedHashMap.
+        Optional<String> sortProperty = Optional.empty();
```

Comment:
Since sortProperty and sortDirection are always used together, prefer to store them both in a single helper object, e.g.

```
static class SortingParameters {
  private String sortBy;
  private Query.SortDirection direction = Query.SortDirection.ASCENDING;

  String sortBy() { return sortBy; }
  Query.SortDirection direction() { return direction; }
}
```

(this way you also only pass a single optional parameter to `listSimulations()`, *and* you don't need to have separate special cases when `sortDirection` is specified explicitly or not.)